### PR TITLE
Update email-settings-android.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a pre req as Chrome app needs to be installed in the work profile to enable modern auth for Gmail Per issue##2892 and support

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/configuration/email-settings-android.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ As an Intune administrator, you can create and assign email settings to Android
 
 ## Before you begin
 
-Create an [Android device administrator Email device configuration profile](email-settings-configure.md).
+- Create an [Android device administrator Email device configuration profile](email-settings-configure.md).
+- Install Chrome App in the work profile to enable modern auth for Gmail.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/email-settings-android.md:42]:
```suggestion
- Install the Chrome app in the work profile to enable modern auth for Gmail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MandiOhlinger Please review and signoff this PR. Thanks

**Comment by MandiOhlinger** [comment]
Closing PR w/o merging. Not sure this requirement applies to Android device administrator.

**Pull request closed without merging**